contracts/BC_fusion/StakeHub.sol

```diff
@@ -863,4 +868,8 @@ contract StakeHub is System {
             }
         }
     }
```

Comment:
We should do sync after `downtimeSlash`, `maliciousVoteSlash` and `doubleSignSlash`

Reply:
fixed